Tests/Form/Type/BooleanTypeTest.php

```diff
@@ -21,14 +21,11 @@ class BooleanTypeTest extends TypeTestCase
 {
     public function testBuildForm()
     {
-        // NEXT_MAJOR: Hack for php 5.3 only, remove it when requirement of PHP is >= 5.4
-        $that = $this;
-
         $formBuilder = $this->getMockBuilder('Symfony\Component\Form\FormBuilder')->disableOriginalConstructor()->getMock();
         $formBuilder
             ->expects($this->any())
             ->method('add')
-            ->will($this->returnCallback(function ($name, $type = null) use ($that) {
+            ->will($this->returnCallback(function ($name, $type = null) use ($this) {
```

Comment:
does it required to pass `$this` via `use`. Looks like it can be safety removed on current php versions